src/components/Create/components/pages/FundingTarget/hooks/FundingTargetForm.ts

```diff
@@ -17,36 +20,38 @@ export const useFundingTargetForm = () => {
   const [form] = Form.useForm<FundingTargetFormProps>()
   const [distributionLimit, setDistributionLimit] =
     useEditingDistributionLimit()
+  const { fundingTargetSelection } = useAppSelector(
+    state => state.editingV2Project,
+  )
   useDebugValue(form.getFieldsValue())
 
   const initialValues: FundingTargetFormProps | undefined = useMemo(() => {
-    if (!distributionLimit) {
-      return undefined
-    }
+    const selection = fundingTargetSelection
 
     const currency =
-      distributionLimit.currency === V2V3_CURRENCY_ETH ? 'eth' : 'usd'
+      distributionLimit?.currency === V2V3_CURRENCY_USD ? 'usd' : 'eth'
+    let amount: CurrencySelectInputValue = { amount: '0', currency }
 
-    if (distributionLimit.amount.eq(MAX_DISTRIBUTION_LIMIT)) {
-      return {
-        targetSelection: 'infinite',
-        amount: { amount: undefined, currency },
-      }
-    } else {
-      return {
-        targetSelection: 'specific',
-        amount: {
-          amount: fromWad(distributionLimit.amount),
+    if (distributionLimit) {
+      if (distributionLimit.amount.eq(MAX_DISTRIBUTION_LIMIT)) {
+        amount = { amount: '0', currency }
```

Comment:
**question** just checkin we def want to set amount to '0' when dist limit is infinite